core/domain/draft_upgrade_services.py

```diff
@@ -135,8 +135,12 @@ def _convert_html_in_draft_change_list(
                 if 'choices' in new_value.keys():
                     for value_index, value in enumerate(
                             new_value['choices']['value']):
-                        new_value['choices']['value'][value_index] = (
-                            conversion_fn(value))
+                        if isinstance(value, dict) and 'html' in value:
```

Comment:
@kevintab95 It would be great if you can leave an inline github-comment explaining why are we doing this change and which issue it fixes?

Reply:
This fixes https://github.com/oppia/oppia/issues/13630. It was erroring due to incorrect handling of cust args when converting HTML content in the draft migration function. We seem to assume that "choices" will always be a string, but that is not always the case. I've made this more determinate now.

Reply:
I see, thanks for the explanation! :)